.github/workflows/autotools.yml

```diff
@@ -60,41 +83,69 @@ jobs:
            git clone https://github.com/egorpugin/tessdata tessdata_unittest
            cp tessdata_unittest/fonts/* test/testing/ 
            mv tessdata_unittest/* ../
+           
+    - name: List languages in different test tessdata-dir
+      run: |
+           tesseract  --list-langs --tessdata-dir ../tessdata
+           tesseract  --list-langs --tessdata-dir ../tessdata_best
+           tesseract  --list-langs --tessdata-dir ../tessdata_fast
+        
+    - name: Run Tesseract on test images in different languages
+      run: |
+           tesseract test/testing/phototest.tif - --oem 1  --tessdata-dir ../tessdata
+           tesseract test/testing/raaj.tif - -l hin --oem 1   --tessdata-dir ../tessdata
+           tesseract test/testing/viet.tif - -l vie --oem 1   --tessdata-dir ../tessdata
+           tesseract test/testing/hebrew.png - -l heb --oem 1   --tessdata-dir ../tessdata
+           tesseract test/testing/eurotext.tif - -l fra --oem 1 --tessdata-dir ../tessdata_best
+           tesseract test/testing/arabic.tif - -l ara --oem 1 --psm 6  --tessdata-dir ../tessdata
+
+    - name: Run Tesseract basicapitest
+      run: |
+           export CXXFLAGS="-std=c++11"
+           cd basicapitest
```

Comment:
This directory does not exist. Nor does basicapitest.cpp.

Reply:
It is in my fork of tesseract. It is a copy of the basic api example from tessdoc.

Actually I wanted to check with you whether the cpp file should be uploaded to the test repo in testing directory and used from there rather than placed in a new directory.

Reply:
Please see if this is ok
https://github.com/tesseract-ocr/test/pull/20

I will change the CI workflow accordingly.